app/extensions/ember-table/template.hbs

```diff
@@ -0,0 +1,8 @@
+<table class="ui table">
```

Comment:
@CosmicCoder96 Can you please also make it unstackable, As it will loose headers in `Mobile View`?